### DISCUSSION TITLE
[TS80P] Flashing via vfat appears to be working in Linux 5.19.3

### DISCUSSION THREAD

**Discussion by froyomuffin** [Ralim/IronOS#1393] [General]
Hey folks, I recently ran into some issues when trying to flash 2.19 onto a TS80P under Arch (5.19.3) using the recommended `msdos` process detailed [here](https://github.com/Ralim/IronOS/blob/dev/Documentation/Flashing.md#linux).

As a last resort, I tried the same procedure under `vfat` (automount) and, to my surprise, it worked!

Has anyone else stumbled upon this? :)

**Comment by wildblue** [comment]
i had problems flashing 2.19 under ubuntu onto my mhp30. normally i would need the hexfile only, but trying to flash that file always resulted in the .err condition.
only afteri uploaded both files to the device (.hex and .dex) i got a successful flash done. it resulted in an error for the dex file and success for the hex file.

i did not alter the device partition settings. ubuntu 18 lts with xfce as desktop manager.

curious, i know...

**Reply by froyomuffin** [reply]
Oh my that is strange. I wasn't aware that the .dex files were needed for flashing at all o.o

I had a really strange experience with the 2.19 and TS80P where I got a successful flash .rdy file but the firmware wasn't actually flashed.

I feel like DFU on these devices is powered by black magic.
